package.json

```diff
@@ -68,7 +68,7 @@
   },
   "peerDependencies": {
     "@edx/frontend-platform": "^1.8.0",
-    "@edx/paragon": "^7.0.0",
+    "@edx/paragon": "^12.8.0",
```

Comment:
I think we need this to stay a range - it should include the lowest version of paragon that still works, i.e., something like:

`"@edx/paragon": ">= ^7.0.0 < 13.0.0"`

That'd be inclusive of the old version number and anything 12.x